pages/_app.js

```diff
@@ -97,8 +123,20 @@ function Main({ component: Component }) {
 
 export default function MyApp({ Component }) {
     return (
-        <PrimeReactProvider>
-            <Main component={Component} />
-        </PrimeReactProvider>
+        <>
```

Comment:
Should we not render this script and run this code in Development mode?  No sense in collecting analytics on local dev right?